scripts/qa_crowbarsetup.sh

```diff
@@ -59,6 +59,8 @@ case "$cloud" in
         want_ipmi=true
     ;;
     d2)
+        nodenumber=2
+        nodeipmilist="10.162.186.163 10.162.186.169"
```

Comment:
I think, we just need to change the 169 to 164 once manually and then the generic IPMI code should work fine.